src/nemos/observation_models.py

```diff
@@ -1165,3 +1165,296 @@ def estimate_scale(
             The DOF of the residuals.
         """
         return jnp.ones_like(jnp.atleast_1d(y[0]))
+
+
+class NegativeBinomialObservations(Observations):
+    r"""
+    A Negative Binomial model for overdispersed count data using mean-dispersion parameterization.
+
+    This model represents a Negative Binomial distribution [3]_ commonly used to model overdispersed
+    count data [4]_ [5]_ (i.e., data where the variance exceeds the mean), which cannot be captured by a
+    standard Poisson model. The distribution is parameterized by the predicted mean rate
+    (:math:`\mu`) and a fixed dispersion parameter (:math:`\phi`) or `scale` of the model.
+
+    **Important:** the scale parameter must be estimated from the data for accurately capturing the
+    dispersion. In the context of NeMoS GLM, estimation can be achieved by cross-validating the
+    ``scale`` parameter. One may use scikit-learn
+    `GridSearchCV <../../how_to_guide/plot_06_sklearn_pipeline_cv_demo.html>`_
```

Comment:
This link works, but would be more future proof to use a proper sphinx link (which I think you can do from docstring), by linking to the explicit anchor `sklearn-how-to`